services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/EntityManager.java

```diff
@@ -454,6 +454,17 @@ public Entity getEntity(UserInfo user, String entityId)
 	 * @return
 	 */
 	public JsonSchemaObjectBinding bindSchemaToEntity(UserInfo userInfo, BindSchemaToEntityRequest request);
+	
+	/**
+	 * Bind a JSON schema to an Entity and send notification messages if sendNotificationMessage is true.
+	 * If sendNotificationMessage is false, performs the bind but does not send notification messages.

```

Comment:
Note:  A notification message is sent to trigger the re-validation of all effected objects when this method is called.